contracts/erc1155/MintableERC1155.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC1155/presets/ERC1155PresetMinterPauser.sol";
+
+import "@openzeppelin/contracts/utils/Context.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+
+contract MintableERC1155 is ERC1155, Ownable {
+    mapping(uint256 => string) public _tokenURIs;
+
+    event Airdrop(address caller, uint256 tokenId, address[] recipients, uint256 amount);
+
+    // solhint-disable-next-line
+    constructor() ERC1155("") {}
+
+    /**
+     * @dev Creates `amount` new tokens for `recipients` of token type `tokenId`.
```

Comment:
Nit, explanation of function should be `@notice` not `@dev`. Use `@dev` for caller warnings like the out of gas limit